.nvmrc

```diff
@@ -0,0 +1 @@
+10.8.0
```

Comment:
I should definitely update this, but don't want to fuss around with Node versions right now :sweat_smile: